### DISCUSSION TITLE
How to stop/start/update containers or trigger the daily backup from a script externally?

### DISCUSSION THREAD

**Discussion by cchalifo999** [nextcloud/all-in-one#1349] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. sudo docker exec -it -e 'DAILY_BACKUP=1 START_CONTAINERS=1 AUTOMATIC_UPDATES=1'  nextcloud-aio-mastercontainer /daily-backup.sh
2.
3.

#### Other valuable info 
This is wat i truyed but the containers are not starting

**Comment by Zoey2936** [comment, marked as answer]
try:
`sudo docker exec -it -e 'DAILY_BACKUP=1' -e 'START_CONTAINERS=1' -e 'AUTOMATIC_UPDATES=1' nextcloud-aio-mastercontainer /daily-backup.sh`

**Comment by cchalifo999** [comment]
> sudo docker exec -it -e 'DAILY_BACKUP=1' -e 'START_CONTAINERS=1' -e 'AUTOMATIC_UPDATES=1' nextcloud-aio-mastercontainer /daily-backup.sh

That worked it started all the conatiners execpt the nextcloud_aio_mastercontainer

**Reply by szaimen** [reply]
The mastercontainer is already running since otherwise you would not have been able to run this command successfully
